core/trino-main/src/test/java/io/trino/server/security/oauth2/TestOAuth2WebUiAuthenticationFilter.java

```diff
@@ -240,7 +240,7 @@ public void testTokenFromTrustedClient()
     }
 
     @Test
-    @Flaky(issue = "https://github.com/trinodb/trino/issues/6223", match = OAUTH2_COOKIE + " is missing")
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/6223", match = "^" /* the test is flaky in a variety of ways */)
```

Comment:
```
/* the test is flaky in a variety of ways due the usage of Selenium */
```

Reply:
i don't know why and don't feel like adding information that i do not have.
if you have more to add, maybe you or @lukasz-walkiewicz want to take it over?

Reply:
ok, you can close this PR then.